framework/langchain/conversational_memory.ipynb

```diff
@@ -0,0 +1,359 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# CrateDB Chat Message History\n",
+    "\n",
+    "This notebook demonstrates how to use the `CrateDBChatMessageHistory`\n",
+    "to manage chat history in CrateDB, for supporting conversational memory."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "f22eab3f84cbeb37"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "Install required packages."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [
+    "#!pip install 'crate[sqlalchemy]' 'langchain[openai]'"
```

Comment:
Unless the adapter is upstreamed and released, we will need to call out the installation command differently.

Reply:
The cell can be written like this, so it will also refer to the accompanied `requirements.txt` file. 👍 
```
   "source": [
    "#!pip install -r requirements.txt"
   ],
```